src/praisonai/praisonai/cli/commands/config.py

```
@@ -384,3 +446,155 @@ def config_doctor():
+        # Check schema validity
+        from ..configuration.resolver import ResolvedConfig
+        
+        try:
+            if isinstance(data, dict):
+                validated = ResolvedConfig.from_dict(data)
+                output.print_success(f"✓ Configuration is valid: {config_path}")
+            else:
+                output.print_error(f"Configuration must be a dictionary/object: {config_path}")
+                raise typer.Exit(1)
+                
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)